spark/src/main/scala/org/apache/spark/sql/delta/coordinatedcommits/InMemoryCommitCoordinator.scala

```diff
@@ -206,6 +206,17 @@ class InMemoryCommitCoordinator(val batchSize: Long)
   }
 
   override def semanticEquals(other: CommitCoordinatorClient): Boolean = this == other
+
+  def removeCommitTestOnly(
```

Comment:
given it got moved to production space, should we make at least private[tahoe]?

Reply:
Yes, limited the access to private[delta]